### PR TITLE
Feature/zoom to cluster

### PR DESCRIPTION
**Pull request by christian-rli**
Uses new layer for state and zooms all the way to cluster. Works with Kano State as an example. Updating layer like for the grid still needs to be implemented.

### CODE REVIEW

File: app/static/js/layers.js [hunk 1]
```diff
@@ -431,6 +432,6 @@ let ogclustersTileLayer = L.vectorGrid.protobuf("https://tile.rl-institut.de/dat
 ogclustersTileLayer.on("click", function (event) {
   map.options.maxZoom = 19;
   village_button_fun();
-  map.flyTo([event.latlng.lat, event.latlng.lng], 14);
```
Comment (Bachibouzouk) [inline app/static/js/layers.js:436]: You don't include the 14 in the change? was it the zoom level?

File: app/static/js/layers.js [hunk 2]
```diff
@@ -361,7 +361,7 @@ let vecTileLayer = L.vectorGrid.protobuf("https://tile.rl-institut.de/data/nesp/
 });
 
 // Vector tiles layer with off-grid-clusters (remotely mapped villages). Contains layer 'OGClusters'. columns: cluster_offgrid_id, area_km2, building_count, large_building_count, percentage_large_building, building_area_km2, building_count_density_perkm2, percentage_building_area, grid_dist_km
-let ogclustersTileLayer = L.vectorGrid.protobuf("https://tile.rl-institut.de/data/nesp2_offgrid_clusters/{z}/{x}/{y}.pbf", {
+let ogclustersTileLayer = L.vectorGrid.protobuf("https://tile.rl-institut.de/data/nesp2_state_offgrid_clusters_kano/{z}/{x}/{y}.pbf", {
```
Comment (Bachibouzouk) [inline app/static/js/layers.js]: Don't you want to make it state specific? Why only Kano?

**Review by Bachibouzouk — COMMENTED** [review]
What is the sequence of steps I should do in order to test the all-the-way zoomin?

File: app/static/js/layers.js (hunk 1 above)
Comment (christian-rli) [reply app/static/js/layers.js:436]: Yes it was. It's now chosen automatically by the bounds

File: app/static/js/layers.js (hunk 2 above)
Comment (christian-rli) [reply app/static/js/layers.js]: Working on being state specific like the grid layer

**Comment by Bachibouzouk** [comment]
@christian-rli - I see the cluster blue area but not the buildings in there, is it normal?

**Comment by christian-rli** [comment]
> What is the sequence of steps I should do in order to test the all-the-way zoomin?

I also removed the geojson layer in order to click. still needs to be worked out.

>@christian-rli - I see the cluster blue area but not the buildings in there, is it normal?

Yes, gray osm does not have up to date tiles. If you switch to hot as a baselayer you can see the buildings. this should happen automatically later on

**Comment by christian-rli** [comment]
interactive:false seems to be the right setting, but it only works for the entire layer, not single objects in it. Moving away from this now.

https://stackoverflow.com/questions/26216926/leaflet-geojson-layer-feature-group-mouse-pointer

**Comment by christian-rli** [comment]
This works apart from the geojson layer on top. Merging so I won't collect PRs

**Pull request merged**